CMakeLists.txt

```diff
@@ -59,12 +55,23 @@ if(MSVC)
     set(CMAKE_DEBUG_POSTFIX "d")
 endif()
 
+# Build position independent code
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+# Disable C and C++ compiler extensions
+set(CMAKE_C_EXTENSIONS OFF)
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
Same as above 😄 . I would add something like

> C/CXX_EXTENSIONS are ON by default to allow the compilers to use extended variants of the C/CXX language. However, this could expose cross-platform bugs in user code or in the headers of third-party dependencies and thus it is strongly suggested to turn extensions off.

which is an elaborated version from CMake documentation itself.